Framework/DataHandling/inc/MantidDataHandling/GenerateGroupingPowder.h

```diff
@@ -8,6 +8,9 @@
 
 #include "MantidAPI/Algorithm.h"
 #include "MantidDataHandling/DllConfig.h"
+#include "MantidDataObjects/GroupingWorkspace.h"
+
+using namespace Mantid::DataObjects;
```

Comment:
`using namespace` should never exist in a header file. It pollutes the namespace of everything that imports the header. `using` directives in the cpp file are fine.